src/components/Profile/components/PersonalInfoList/index.jsx

```diff
@@ -247,6 +248,30 @@ const PersonalInfoList = ({ myProf, profile, isOnline, createSnackbar }) => {
       />
     );
 
+  const plannedGraduationYear =
+    isFacStaff || (isAlumni && !profile.PlannedGradYear?.length) ? null : (
+      <ProfileInfoListItem
+        title={'Planned Graduation Year:'}
+        contentText={
+          myProf && isStudent ? (
+            <Grid container spacing={0} alignItems="center">
+              <Grid item>
+                {!profile.PlannedGradYear?.length ? 'Deciding' : profile.PlannedGradYear}
```

Comment:
```suggestion
                {!profile.PlannedGradYear?.length ? [ACTUAL CLASS YEAR] : profile.PlannedGradYear}
```
I think putting the actual class year here would be significantly more clear than a "deciding" class year

Reply:
Agree, that would be better

Reply:
Just changed so that It will only show for other people if the user ever sets it to a specific year. For your profile it will show a message in the place of "deciding"